featuretools/entityset/entityset.py

```diff
@@ -790,6 +792,7 @@ def normalize_entity(self, base_entity_id, new_entity_id, index,
             new_entity_id,
             new_entity_df,
             index,
+            already_sorted=already_sorted,
```

Comment:
Can we update the `normalize_entityset` tests to confirm that created time indexes are sorted